Gemfile.lock

```diff
@@ -254,8 +257,9 @@ DEPENDENCIES
   jbuilder (~> 2.7)
   listen (~> 3.2)
   pg (~> 1.0)
-  puma (~> 4.3)
-  rails (~> 6.1.2)
+  puma (~> 4.1)
```

Comment:
i think we want this to remain 4.3

Reply:
when i bundle install locally i also get a `stimulus-rails` added on line 270. can you try running bundle install on this again and commiting those changes?

Reply:
ok....tried again.....